src/components/TaskStatusFilters/index.tsx

```diff
@@ -52,7 +31,7 @@ export const TaskStatusFilters: React.FC<Props> = ({
           <TreeSelect
```

Comment:
Is this only here for the VersionRestartModal since we don't need it for the task tables anymore?

Reply:
that's right. only the VersionRestartModal is using this componenet